src/index.ts

```diff
@@ -14,12 +14,12 @@ checkCWD()
 
 export * as Args from './args'
 export * as Errors from './errors'
-export * as Flags from './flags'
 export * as Interfaces from './interfaces'
+export * as Flags from './flags'
 export * as Parser from './parser'
 export * as ux from './cli-ux'
 export {CommandHelp, HelpBase, Help, loadHelpClass} from './help'
-export {Config, toCached, Plugin, tsPath} from './config'
+export {Config, Plugin, tsPath} from './config'
```

Comment:
I doubt anyone was using it, but we had toCached exported and now it's not.  Breaking change for your list?

Reply:
Yes, this was intentional. I will add it to the list of breaking changes